src/Mod/Draft/draftobjects/drawingview.py

```diff
@@ -0,0 +1,141 @@
+# ***************************************************************************
+# *   Copyright (c) 2009, 2010 Yorik van Havre <yorik@uncreated.net>        *
+# *   Copyright (c) 2009, 2010 Ken Cline <cline@frii.com>                   *
+# *   Copyright (c) 2020 FreeCAD Developers                                 *
+# *                                                                         *
+# *   This program is free software; you can redistribute it and/or modify  *
+# *   it under the terms of the GNU Lesser General Public License (LGPL)    *
+# *   as published by the Free Software Foundation; either version 2 of     *
+# *   the License, or (at your option) any later version.                   *
+# *   for detail see the LICENCE text file.                                 *
+# *                                                                         *
+# *   This program is distributed in the hope that it will be useful,       *
+# *   but WITHOUT ANY WARRANTY; without even the implied warranty of        *
+# *   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the         *
+# *   GNU Library General Public License for more details.                  *
+# *                                                                         *
+# *   You should have received a copy of the GNU Library General Public     *
+# *   License along with this program; if not, write to the Free Software   *
+# *   Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  *
+# *   USA                                                                   *
+# *                                                                         *
+# ***************************************************************************
+"""This module provides the object code for the Draft DrawingView object.
+"""
+## @package drawingview
+# \ingroup DRAFT
+# \brief This module provides the object code for the Draft DrawingView object.
+
+import math
+
+from PySide.QtCore import QT_TRANSLATE_NOOP
+
+import FreeCAD as App
+
+import DraftVecUtils
+
+import getSVG as svg
+getSVG = svg.getSVG
```

Comment:
This is a bit convoluted. It's simpler to use
```py
from getSVG import getSVG
```

Reply:
I know, but it was like this, and I did not know if some other moduels used some functions from Draft.svg.xxx  .... if you think so we can simplify it.

Reply:
Well, I advice making the change, and then, of course, testing if there are problems elsewhere. But I think `Draft.getSVG` is what is used by the Arch classes.